src/app/containers/Timestamp/relativeTimestamp.js

```diff
@@ -2,7 +2,10 @@ const constructTimestamp = timeDiffMins => {
   let timeDiff;
   let unit;
 
-  if (timeDiffMins >= 60) {
+  if (timeDiffMins >= 1440) {
```

Comment:
Is there a reason for adding support for days?

Reply:
Days support was added so that relativeTimestamp() could be generic enough to capture any time period e.g. 1 hour ago, 23 days ago